protocol/app/app.go

```diff
@@ -1073,6 +1101,7 @@ func New(
 		upgradetypes.ModuleName,
 		ibctransfertypes.ModuleName,
 		ratelimitmoduletypes.ModuleName,
+		icatypes.ModuleName,
```

Comment:
No, since the [InitGenesis](https://github.com/cosmos/ibc-go/blob/640b13997b6199498ab4473d4e8e999679a4b7a7/modules/apps/27-interchain-accounts/module.go#L154) implementation is independent of other modules.